packages/staking/src/features/store/delegationPortfolioStore/useDelegationPortfolioStore.ts

```diff
@@ -26,6 +27,27 @@ const defaultState: DelegationPortfolioState = {
   viewedStakePool: undefined,
 };
 
+// If percentages add up to 100, normalize them. Otherwise, round them to N decimal places
+const adjustOnchainPercentages = <K extends string, T extends { [key in K]: number }>({
```

Comment:
in line with my other [suggestion](https://github.com/input-output-hk/lace/pull/611/files#r1347649019)  I'd use the term "sanitize" 

```suggestion
const sanitizeOnchainPercentages = <K extends string, T extends { [key in K]: number }>({
```